models/user-model.js

```diff
@@ -9,8 +10,7 @@ const UserSchema = new Schema({
   confirmedEmail: {type: Boolean, default: false},
   emailConfirmationLink: {type: String},
   memberSince: { type: Date},
-  // TODO: add model
-  friends : [String],
+  friends : [ FriendSchema ],
```

Comment:
https://medium.com/@brandon.lau86/one-to-many-relationships-with-mongodb-and-mongoose-in-node-express-d5c9d23d93c2